mlflow/metrics/genai/genai_metric.py

```diff
@@ -262,11 +262,27 @@ def make_genai_metric_from_prompt(
             greater_is_better=True,
         )
 
+        data = pd.DataFrame(
+            {
+                "input": ["Where is the capital of France."],
+                "ground_truth": ["Paris"],
+                "output": ["The capital of France is Paris."],
+            }
+        )
+
+        mlflow.evaluate(
+            data=data,
+            targets="ground_truth",
+            predictions="output",
+            evaluators="default",
+            extra_metrics=[metric],
+        )
     """
     import numpy as np
 
     prompt_template = PromptTemplate([judge_prompt, _PROMPT_FORMATTING_WRAPPER])
     allowed_variables = prompt_template.variables
+    additional_variables = list(allowed_variables - {"predictions", "targets"})
```

Comment:
Was `targets` always an unused variable in the kwargs processing from before?

Reply:
Good catch, I just forgot to remove "targets"